pkg/provider/azure_loadbalancer_repo.go

```diff
@@ -410,3 +410,41 @@ func isNICPool(bp *armnetwork.BackendAddressPool) bool {
 	}
 	return false
 }
+
+// checkRemoveOutdatedBasicLoadBalancers removes outdated basic load balancers
+// when the loadBalancerSkus is `Standard`. It ensures the backend pool of the basic
+// load balancer is empty before deleting the load balancer.
+func (az *Cloud) checkRemoveOutdatedBasicLoadBalancers(
```

Comment:
may rename to `cleanupBasicLoadBalancer()` for clear?

Reply:
done